### PR TITLE
[R/es] Added Spanish translation for R

### PR DESCRIPTION
**Pull request by deivuh**

### CODE REVIEW

File: r.html.markdown [hunk 1]
```diff
@@ -2,7 +2,7 @@
 language: R
 contributors:
     - ["e99n09", "http://github.com/e99n09"]
-    - ["isomorphismes", "http://twitter.com/isomorphisms"]
+    - ["isomorphismes", "http://twitter.com/isomorphisms"]
```
Comment (vendethiel) [inline r.html.markdown]: this is probably the reason for the merge conflict, can you please fix it?

**Comment by deivuh** [comment]
Done. Please review it.

**Comment by vendethiel** [comment]
Yay, thanks! Sorry for the delay.

**Comment by vendethiel** [comment]
:ship:

**Pull request merged**